website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, host_page, host_name, mode, options) {
```

Comment:
I see that mode is used to conditionally apply the mousemove and mouseup listeners, but why not expose that in `options` rather than passing in `mode` (which is a concern of the Comment view-model)?